README.md

```diff
@@ -48,9 +48,9 @@ npm install --save-dev redux-devtools
 
 ### The Gist
 
-The whole state of your app is stored in an object tree inside a single *store*.  
-The only way to change the state tree is to emit an *action*, an object describing what happened.  
-To specify how the state tree is transformed by the actions, you write pure *reducers*.
+The whole state of your app is retained in a single object that we will call *the store*.  
+The only way to alter the store is to emit an *action*, an object describing what is happening.  
```

Comment:
I don't think one can “alter the store”, it's just a storage. One can alter the state tree.